mantra/config/__init__.py

```diff
@@ -1,3 +1,4 @@
 from .bot_config import bot_config
+from .database import db_config
 
 __all__ = ["bot_config"]
```

Comment:
Add `db_config` to the `__all__` variable

Reply:
Will do!